stdlib/io.pyi

```diff
@@ -146,10 +149,33 @@ class TextIOBase(IOBase):
     def readlines(self, __hint: int = -1) -> list[str]: ...  # type: ignore[override]
     def read(self, __size: int | None = ...) -> str: ...
 
+@type_check_only
+class _WrappedBuffer(Protocol):
+    @property
+    def name(self) -> str: ...
+    @property
+    def closed(self) -> bool: ...
+    def read(self, size: int = ...) -> bytes: ...
```

Comment:
Are there any instances of `size` being passed as a keyword argument? If not, the parameter in the protocol should be pos-only:

```suggestion
    def read(self, size: int = ..., /) -> bytes: ...
```